### PR TITLE
Community tweak session

### PR DESCRIPTION
**Pull request by AndrewBestbier**
The tweaks done in this branch can be seen [here](https://trello.com/c/gOz0hRso).

Alt Text

### CODE REVIEW

File: shared/components/ConferencePartners/index.js [hunk 1]
```diff
@@ -21,27 +21,26 @@ const ConferencePartners = (props) => {
         <div className="content">
           <p>
             React London 2017 is possible due to the kind support
-            of <strong> our amazing partners</strong>, who help us keep
-            tickets affordable.
+            of <strong> our amazing partners</strong>.
```
Comment (lpil) [inline shared/components/ConferencePartners/index.js]: Space on both sides of the tag here?

File: shared/components/EventSchedule/index.scss [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
+  border-right: none;;
   &__schedule {
     list-style-type: none;
     border-left: $dash-border;
     margin: 0 30px 40px 80px;
     position: relative;
   }
 }
+
+@media (min-width: 768px) {
```
Comment (lpil) [inline shared/components/EventSchedule/index.scss]: Use the media query variables instead :)

File: shared/components/NavigationBar/index.scss [hunk 3]
```diff
@@ -19,7 +20,7 @@
         color: $conference-color;
         text-decoration: none;
         &:hover {
-          background: $conference-color;
```
Comment (lpil) [inline shared/components/NavigationBar/index.scss]: Use the `$white` variable

File: shared/components/TicketStatus/index.scss [hunk 4]
```diff
@@ -19,6 +24,10 @@
     text-decoration: underline;
     color: $grey;
     line-height: 1.4em;
+
+    &:hover {
+      color: #777;
```
Comment (lpil) [inline shared/components/TicketStatus/index.scss]: All colours should be in variables.

File: shared/components/EventSchedule/index.scss [hunk 5]
```diff
@@ -1,8 +1,15 @@
 .EventSchedule {
+  border-right: none;;
```
Comment (lpil) [inline shared/components/EventSchedule/index.scss]: Double semicolon :)

File: shared/components/EventDetails/index.scss [hunk 6]
```diff
@@ -3,6 +3,7 @@
```
Comment (zoeabryant) [inline shared/components/EventDetails/index.scss]: In this files there is the following line:
```
    &-container {
      flex: 1;
```
is flex 1 needed on the container? I understood that a number property is only relevant to the flex-items within a flex container

File: shared/components/EventSchedule/index.scss [hunk 7]
```diff
@@ -1,8 +1,15 @@
 .EventSchedule {
+  border-right: none;
```
Comment (zoeabryant) [inline shared/components/EventSchedule/index.scss]: is this needed? As this is mobile first I can't see why we'd need to override line 13

File: shared/components/JobsSection/index.scss [hunk 8]
```diff
@@ -78,6 +78,9 @@
     &:first-child {
       padding-top: 20px;
     }
+    &__title {
+      margin-top: 10px;
```
Comment (zoeabryant) [inline shared/components/JobsSection/index.scss]: is there is already padding on the parent element of `__title` -  can we add 10px to this instead? It gets annoying if spacing that visually affects the same space is actually set in two different places.

**Comment by zoeabryant** [comment]
Added a few comments just in regards to css best practice stuff - these do not block the PR from going through.

**Pull request merged**